doc/filters/escape.rst

```diff
@@ -8,7 +8,7 @@
 .. versionadded:: 1.14.0
     The ability to define custom escapers was added in Twig 1.14.0.
 
-The ``escape`` filter escapes a string for safe insertion into the final
+The ``escape`` filter escapes a string for safe insertion into the final HTML
```

Comment:
Using HTML here seems misleading to me. It may give the feeling that `escape` is only useful if your template produces HTML.

Reply:
I've tried something else in #2980. Does that work better for you?